cap_client/utils.py

```diff
@@ -82,6 +82,18 @@ def load_json(ctx, param, value):
             raise BadParameter('Not a valid JSON.')
 
 
+def load_value(ctx, param, value):
+    if value is not None:
+        try:
+            return json.loads(value)
+        except (KeyError, ValueError):
+            try:
+                val = float(value)
+                return int(val) if val.is_integer() else val
```

Comment:
why do you cast integers twice?

this is not gonna work when you want to pass a string, that can be also casted to a number, for examplee we have this for some final state objects, where user can say `2`, `<=3` etc, so we keep this field as a string, but some of the values could be casted to int.